src/OpenLoco/src/Objects/ObjectManager.cpp

```diff
@@ -309,6 +501,7 @@ namespace OpenLoco::ObjectManager
     static void callObjectLoad(const LoadedObjectHandle& handle, Object& obj, std::span<const std::byte> data, DependentObjects* dependencies = nullptr)
     {
         return visitObject(handle.type, obj, [&](auto&& obj) {
+            std::memcpy(obj, data.data(), sizeof(*obj));
```

Comment:
todo: This needs to become a field by field reader in each of the load functions if we ever want to be non-x86.